Cargo.toml

```diff
@@ -149,7 +149,8 @@ features = [
     'Node',
     'PointerEvent',
     'Window',
-    'WheelEvent'
+    'WheelEvent',
+    "HtmlScriptElement",
```

Comment:
This should go into `dev-dependencies`.
Also apparently we are using `'` instead of `"`.